### PR TITLE
improve: improve sample app functionality (SDKCF-5949)

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
* Adde tab bar for Tooltip testing
* Added a dedicated tab for custom events
* Added ipa generation lane

## Links
SDKCF-5949

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data **before every commit**, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -92,4 +92,21 @@ platform :ios do
       xcargs: "CODE_SIGN_IDENTITY='' CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO ENABLE_BITCODE=NO RIAM_CONFIG_URL=#{config_url}",
     )
   end
+
+  desc "Create and sign ipa from generated xcarchive artifact"
+  lane :create_ipa do |options|
```
Comment (imaginaris) [inline fastlane/Fastfile:97]: to be confirmed

Comment (imaginaris) [reply fastlane/Fastfile:97]: Confirmed. The lane can be used locally and on Bitrise

File: Sample/CustomEventViewController.swift [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            return CustomAttribute(withKeyName: name,
+                                   withTimeInMilliValue: value.integerValue)
+
+        default:
+            return nil
+        }
+    }
+
+    fileprivate func showAttributeTypeSelection(callback: @escaping (String) -> Void) {
+
```
Comment (malayli) [inline Sample/CustomEventViewController.swift]: You can remove this empty line.

File: Sample/Helpers/String+Conversion.swift [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    var hasDoubleValue: Bool {
+
+        Double(self) != nil
+
+    }
+
+    var doubleValue: Double {
+
+        (self as NSString).doubleValue
+
```
Comment (malayli) [inline Sample/Helpers/String+Conversion.swift]: There are many empty lines. Is it necessary?

Comment (imaginaris) [reply Sample/Helpers/String+Conversion.swift]: oh wow. I don't know how that happened. Fixing...

File: Sample/ViewController.swift [hunk 4]
```diff
@@ -1,8 +1,13 @@
 import UIKit
-import RInAppMessaging
+@testable import RInAppMessaging
```
Comment (malayli) [inline Sample/ViewController.swift:2]: @imaginaris `@testable` is needed in order to use an internal API of `RInAppMessaging`?

Comment (imaginaris) [reply Sample/ViewController.swift:2]: Yes. It's not a very clean solution but I wanted to provide a proper UX for testers. There's no other way to get information about initialization status of the SDK without modifying public API. But still I have mixed feelings

**Review by malayli — APPROVED** [review]

**Pull request merged**